rllib/algorithms/ppo/ppo.py

```diff
@@ -376,57 +402,122 @@ def get_default_policy_class(
             return PPOTF2Policy
 
     @override(Algorithm)
-    def training_step(self) -> ResultDict:
-        use_rollout_worker = self.config.env_runner_cls is None or issubclass(
-            self.config.env_runner_cls, RolloutWorker
-        )
+    def training_step(self):
```

Comment:
Disseminate these two stacks now for better readability (avoid too many if-blocks inside `training_step`).

Old- and hybrid stacks will be deprecated soon anyways.